src/components/calcite-action-bar/calcite-action-bar.e2e.ts

```diff
@@ -151,4 +151,23 @@ describe("calcite-action-bar", () => {
       </calcite-action-group>
     </calcite-action-bar>
     `));
+
+  it("should focus on toggle button", async () => {
+    const page = await newE2EPage({
+      html: `<calcite-action-bar>
+      <calcite-action-group>
+        <calcite-action text="Add" icon="plus"></calcite-action>
+      </calcite-action-group>
+    </calcite-action-bar>`
+    });
+
+    const tagName = await page.evaluate(async () => {
```

Comment:
Did you try the `focusable` test helper? If not, I think I can add some options to make it work for these tests.

Reply:
yeah it wasn't working correctly

Reply:
The focusable test helper assumes that the focused tag should be the same component, but in this case I want to make sure the action is focused.

The focusable helper would need to be enhanced to take a setFocus argument and an additional tag selector.

Reply:
I'll submit a PR for this. 💪🏼

Reply:
Thank you kind sir! 💪